### PR TITLE
Fix extents for data classification

### PR DESCRIPTION
**Pull request by lyzidiamond**
Turns out my fix before didn't really fix anything! Data extents now change when the year changes.

### CODE REVIEW

File: assets/scripts/main.js [hunk 1]
```diff
@@ -384,6 +385,15 @@ function changeMetric(data) {
     });
 }
 
+function setExtent(msg, data) {
```
Comment (eeeschwartz) [inline assets/scripts/main.js:388]: Doesn't look like the param 'data' gets used, perhaps it is meant to be 'metricData'?

Comment (eeeschwartz) [inline assets/scripts/main.js:388]: perhaps the msg param is extraneous as well?

Comment (lyzidiamond) [inline assets/scripts/main.js:388]: That's some weird PubSub thing I don't understand. It wouldn't run without those params. If you look at the other functions that are called by PubSub, they also have those params and don't use them in their functions...

Comment (eeeschwartz) [inline assets/scripts/main.js:388]: curious indeed.

**Comment by lyzidiamond** [comment]
Related to this, we have some weird outlier data in the Property Values. I am thinking about pulling it out, but am open to discussion. Will open an issue on this now.

**Pull request merged**